app/src/main/java/com/larryhsiao/nyx/MainActivity.java

```diff
@@ -0,0 +1,46 @@
+package com.larryhsiao.nyx;
+
+import android.os.Bundle;
+import androidx.annotation.Nullable;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+import com.larryhsiao.nyx.android.jot.CalendarFragment;
+import com.larryhsiao.nyx.android.jot.JotListFragment;
+import com.larryhsiao.nyx.android.jot.JotMapFragment;
+import com.larryhsiao.nyx.migration.MigrateFromV1;
+import com.silverhetch.aura.AuraActivity;
+
+/**
+ * Entry Activity of Nyx.
+ */
+public class MainActivity extends AuraActivity {
+    private BottomNavigationView navigation;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'navigation' could be replaced by a local variable.](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5029215)